hs_core/templates/pages/genericresource.html

```diff
@@ -975,6 +989,78 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                 </div>
                                 {% endif %}
 
+                                  <!-- ========= CONNECTIONS TAB ========= -->
+                                {% if connections %}
+                                    <div class="tab-pane" id="connectionsTab">
+                                         {% if connections.collected_by %}
+                                                <h4><strong>Belongs to Collections:</strong></h4>
+                                                <hr style="margin-top: 0;margin-bottom: 2px">
+                                                <table id="collected-by-table" class="table-hover table-striped resource-custom-table" width="100%">
+                                                    <thead>
+                                                        <tr>
+                                                            <th>Title</th>
+                                                            <th>Owners</th>
+                                                            <th>Sharing Status</th>
+                                                            <th>My Permission</th>
+                                                        </tr>
+                                                    </thead>
+                                                    <tbody>
+                                                        {% for res in connections.collected_by %}
```

Comment:
@zhiyuli and @dtarb - The entire landing page is just a view of information about a resource. That information can be what is encoded in the science metadata document, but some of it may only exist in the Django database (like the fact that a resource has been added to a collection). I don't see that as a problem.  But, I do think it would be confusing to spread relationships out across multiple tabs. We anticipated this need, we added the "Related Resources" tab specifically for this purpose, and we should use it. It doesn't seem like it should be harder to add this information to an existing tab versus creating a new tab.